src/components/books/IndividualBook.module.css

```diff
@@ -20,8 +22,103 @@
   background-color: transparent;
   padding: 5px 10px;
   font-size: 20px;
-  color: rgb(35, 35, 216);
+  color: rgb(113, 153, 228);
   outline: none;
   border: none;
   cursor: pointer;
 }
+
+.design {
+  margin: 0 2rem 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.design > * {
+  padding: 0 3rem;
+}
+
+.design > *:not(:last-child) {
+  border-right: 1px solid #eee;
+}
+
+.current {
+  font-size: 14px;
+  color: rgb(156, 152, 152);
+  margin-bottom: 2px;
+}
```

Comment:
- [ ] Great job on changing the color of your elements to match that of the provided design however you still need to fix the typography i.e. font-family, size, weight etc to match the provided design. Please login [login to Zeplin](https://app.zeplin.io/login) using this credentials
username: student@microverse.org
password: jb077mL2dL0DfxJ 
and checkout the typography to match the design.

Reply:
i Can't find any font name robotoSlab